app/services/bolognese/readers/base_work_reader.rb

```diff
@@ -58,18 +62,35 @@ def read_work(string: nil, **options)
         @reader_attributes.merge(read_options)
       end
 
+      # Override the parent method as we have other fields
+      def publication_year
```

Comment:
I'm not really excited about overriding this method because it doesn't seem to be following the patterns of bolognese where it's trying to map into the internal data structure.  I'm concerned that this may break other things or that it makes our use of bolognese pointless (which could be okay).  Maybe we could chat about this tomorrow?

Reply:
Hi @cjcolvar, 

I'm more than happy to change this. I've got an idea how I could achieve this, so I'll give that a go and get you to check it out when you're online :)

Reply:
@cjcolvar I've refactored it to not override the method, but set the meta value the method relies on.